samples/bluetooth/broadcast_audio_source/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -1,3 +1,5 @@
+# The LC3 codec uses a large amount of stack. This app runs the codec in the work-queue, hence
+# increase stack size for that thread.
 CONFIG_MAIN_STACK_SIZE=4096
```

Comment:
This comment does apply to the main stack

Reply:
Ditto for a few other places

Reply:
Removed... the sample seems to build and work without this increase.